package.json

```diff
@@ -1745,7 +1745,7 @@
     },
     "scripts": {
         "package": "gulp clean && gulp prePublishBundle && vsce package -o ms-toolsai-jupyter-insiders.vsix",
-        "compile": "tsc -watch -p ./",
+        "compile": "gulp includeBCryptGenRandomExe && tsc -watch -p ./",
```

Comment:
Instead of this, it would be better to do this as part of a `postInstall` step.
Thats where we should be pulling down required dependencies.

Reply:
See existing `postInstall.js`

Reply:
Yeah I have a postinstall step to download the executable into the src dir. THis is copying the executable to the outdir. I guess I could just copy the executable to the outdir as part of the postinstall?